libevmasm/AssemblyItem.h

```diff
@@ -243,6 +260,7 @@ class AssemblyItem
 	size_t m_modifierDepth = 0;
 
 	void setImmutableOccurrences(size_t _n) const { m_immutableOccurrences = _n; }
+	size_t realativeJumpOffset() const;
```

Comment:
```suggestion
	size_t relativeJumpOffset() const;
```

I'd also put it next to `splitForeignPushTag()`, because they're closely related.

Reply:
And there's another typo in: `realtiveJumpTo()` -> `relativeJumpTo()`.